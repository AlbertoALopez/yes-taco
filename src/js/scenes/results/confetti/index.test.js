import React from 'react';
import Confetti from './index.js';
import renderer from 'react-test-renderer';

test('Confetti renders correctly', () => {
  const tree = renderer.create(<Confetti />).toJSON();

  expect(tree).toMatchSnapshot();
});
