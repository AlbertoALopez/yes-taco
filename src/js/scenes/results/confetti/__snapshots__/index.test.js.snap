// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confetti renders correctly 1`] = `
<div
  className="confetti-container"
>
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
  <div
    className="confetti"
  />
</div>
`;
